tests/unit/service/test_info.py

```diff
@@ -683,6 +707,13 @@ async def test_get_target_info(app_container, entity, target):
         has_recipe=bool(target.graph),
         created_at=target.created_at,
         updated_at=target.updated_at,
+        input_data=InputData(
```

Comment:
this feels a little duplicative from the metadata, but this was mentioned in the spec, and hence i'm adding it here. there'll be a bit more information when the forward joins are added i believe.